### PR TITLE
Fixed owner typo

### PR DESCRIPTION
**Pull request by kkhitesh**
Fixes #

## Proposed Changes

  -
  -
  -

### CODE REVIEW

**Comment by Sonichigo** [comment]
Just add #22 to let the issue be closed!

**Comment by kkhitesh** [comment]
Fixed Issue #22

**Comment by Panquesito7** [comment]
> Just add #22 to let the issue be closed!

@kkhitesh, they meant to add this in the first comment. 🙂

**Review by Panquesito7 — APPROVED** [review]
Nice catch! Thanks for your contribution. 😄👍

**Review by Evavic44 — APPROVED** [review]
LGTM. 👍🏾

File: action.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ inputs:
     description: 'GitHub token for repo'
     required: true
   owner-name:
-    description: 'Onwer name is user or organisation'
+    description: 'Owner name is user or organisation'
```
Comment (Vyvy-vi) [inline action.yml:8]: lgtm!

**Review by Vyvy-vi — APPROVED** [review]

**Review by eddiejaoude — APPROVED** [review]
Thanks 👍

**Pull request merged**